lib/public_api_v1.ts

```diff
@@ -30,10 +30,42 @@ import { CreateMessage, GetMessage, GetMessages } from "./controllers/messages";
 import { GetProfile, UpsertProfile } from "./controllers/profiles";
 import { toNonEmptyString } from "./utils/strings";
 
-// Setup Express
+import * as helmet from "helmet";
+import * as csp from "helmet-csp";
+import * as referrerPolicy from "referrer-policy";
 
+// Setup Express
 const app = express();
 
+// Set header `referrer-policy` to `no-referrer`
+app.use(referrerPolicy());
+
+// Set up Content Security Policy
+app.use(
+  csp({
+    directives: {
+      defaultSrc: ["'none'"],
+      upgradeInsecureRequests: true
+    }
+  })
+);
+
+// Set up the following HTTP headers:
+//    strict-transport-security: max-age=15552000; includeSubDomains
```

Comment:
since the actual headers may change with a new version, it makes sense to link to the relevant documentation